pkg/server/catalog/catalog.go

```diff
@@ -166,16 +58,35 @@ type Config struct {
 }
 
 type Repository struct {
-	Catalog
-	catalog.Closer
+	datastore.Repository
```

Comment:
Temporary until datastore plugin artifacts are fully removed?

Reply:
I'm not sure exactly how we want to present the DataStore (e.g. via Catalog or not). If we eventually make storage pluggable, we probably still want the datastore served by the catalog, albeit implemented with a facade over different backing plugins.

So.... TBD?